app/notifiers/publishing_api_notifier.rb

```diff
@@ -65,15 +80,23 @@ def perform(tag_id)
 
   class RootBrowsePageWorker
     include Sidekiq::Worker
-    def perform
-      Services.publishing_api.put_content_item("/browse", RootBrowsePagePresenter.new.render_for_publishing_api)
+    def perform(to_be_published)
```

Comment:
This method and `RootTopicWorker#perform` seem to be duplicating `PublishingAPINotifier#write_content`. Maybe we could extract it out somewhere?